### PR TITLE
Show to the end user that the alternative text is applied or not.

### LINKED ISSUE #12268: Image alternative text presence should be conveyed to the user
## 📝 Provide detailed reproduction steps (if any)

Currently the end user has no idea whether an alternative text is applied to a certain image or not.

1.  Open [full featured example for the latest release](https://ckeditor.com/docs/ckeditor5/35.0.1/examples/builds-custom/full-featured-editor.html).
2.  Focus the first image.
3.  Change alternative text to non-empty value.

### ✔️ Expected result

The fact that the image has an alternative text should be conveyed to the user with UI.

**In addition** to that let's change the button name to "Image text alternative".

### ❌ Actual result

No information is present.

### ❓ Possible solution

IMO just marking the UI button as `isOn` would be sufficient and consistent with how ImageCaption, TableCaption and the alikes are doing this:

For describing link state we even go extra mile. Not only we're setting `isOn` to `true` on link button, but there's this little "status" icon in the top-right corner of the image:

## 📃 Other details

*   Browser: Any
*   OS: Any

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by pszczesniak**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Fix (image): Show to the end user that the alternative text is applied or not. Closes #12268.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-image/tests/imagetextalternative/imagetextalternativeui.js [hunk 1]
```diff
@@ -110,6 +110,16 @@ describe( 'ImageTextAlternativeUI', () => {
 
 			sinon.assert.callOrder( disableCssTransitionsSpy, addSpy, selectSpy, enableCssTransitionsSpy );
 		} );
+
+		it( 'has isOn bound to command\'s value', () => {
+			const button = editor.ui.componentFactory.create( 'imageTextAlternative' );
```
Comment (mmotyczynska) [inline packages/ckeditor5-image/tests/imagetextalternative/imagetextalternativeui.js:115]: It is not necessary to create here this button as we already have a `button` instance that we can re-use, created in the beforeEach block.

**Review by mmotyczynska — COMMENTED** [review]
The fix looks fine, great work :+1: 

The only thing I would change is this unnecessary button instance creation, please check the comment.

File: packages/ckeditor5-image/tests/imagetextalternative/imagetextalternativeui.js (hunk 1 above)
Comment (pszczesniak) [reply packages/ckeditor5-image/tests/imagetextalternative/imagetextalternativeui.js]: Fixed, ty for help.

**Review by mmotyczynska — APPROVED** [review]

**Review by mlewand — APPROVED** [review]
LGTM

**Pull request merged**